src/test/java/StringTest.java

```diff
@@ -0,0 +1,25 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+    @Test
+    void 요구사항1(){
+        assertThat("1,2".split(",")).containsExactly("1","2");
+        assertThat("1".split(",")).containsExactly("1");
+    }
+
+    @Test
+    void 요구사항2(){
+        assertThat("(1,2)".substring(1,4)).isEqualTo("1,2");
+    }
+
+    @Test
+    void 요구사항3(){
+        assertThatThrownBy(() -> {
+            assertThat("abc".charAt(0)).isEqualTo('a');
```

Comment:
```suggestion
    String value = "abc"
    assertThat(value.charAt(0)).isEqualTo('a');
```

테스트 코드에서는 더더욱 클린하게 읽기쉽게 구현하는게 중요하다고 생각해요.
반복되거나 의미를 가지는 값에 대해서, 의도를 드러내는 네이밍으로 추출해보아도 좋겠네요!